pkg/storage/kubernetes.go

```diff
@@ -83,8 +82,8 @@ func (k kubernetesClient) Delete(name string) error {
 	return nil
 }
 
-// ListFromCluster lists pvc based Storage from the cluster
-func (k kubernetesClient) ListFromCluster() (StorageList, error) {
+// List lists pvc based Storage from the cluster
```

Comment:
Are you completely sure you can replace List with ListFromCluster? At first look, they are not doing the same thing